physics/sfcsub.F

```diff
@@ -581,6 +581,7 @@ subroutine sfccycle(lugb,len,lsoil,sig1t,deltsfc                  &
       character*500 fndclm,fndanl
 !
       logical lanom
+      character(len=10) :: message
```

Comment:
I don't think this line is required.

Reply:
Corrected.

Reply:
Thanks. hera is still in maintenance, will start testing when the machine becomes available.